pkg/api/api.go

```diff
@@ -132,6 +140,26 @@ func NewAPI(treeID uint) (*API, error) {
 		return nil, fmt.Errorf("unable get sharding details from sharding config: %w", err)
 	}
 
+	cachedCheckpoints := make(map[int64]string)
+	for _, r := range ranges.GetInactive() {
+		tc := trillianclient.NewTrillianClient(ctx, logClient, r.TreeID)
+		resp := tc.GetLatest(0)
+		if resp.Status != codes.OK {
+			return nil, fmt.Errorf("error with GetLatest(): resp code is %d", resp.Status)
```

Comment:
maybe make this error message a bit clearer and/or wrap resp.Err?

Reply:
done